CONTRIBUTING.md

```diff
@@ -37,8 +37,8 @@ guidelines for Boulder contributions.
   but make sure it's clear why things are being done. Include information in
   your pull request about what you're trying to accomplish with your patch.
 * Avoid named return values. See
-  [#3017](https://github.com/letsencrypt/boulder/pull/3017) for an example of a
-* subtle problem they can cause. Do not include `XXX`s or naked `TODO`s. Use
+  [#3017](https://github.com/letsencrypt/boulder/pull/3017) for an example of a subtle problem they can cause.
```

Comment:
nit: we try to keep this file linewrapped at 80 chars, so I think the right fix is bumping "Do not include..." down, rather than bumping "subtle problem..." up.

Reply:
Fixed, thanks!